packages/link-modules/src/definitions/cart-region.ts

```diff
@@ -0,0 +1,28 @@
+import { Modules } from "@medusajs/modules-sdk"
+import { ModuleJoinerConfig } from "@medusajs/types"
+
+export const CartRegion: ModuleJoinerConfig = {
```

Comment:
comment: Since `region_id` lives on the Cart model, we don't need a link table to associate the two entities, so a read-only link is sufficient.